mappings/1.3.tiny

```diff
@@ -24349,7 +24349,7 @@ c	net/minecraft/unmapped/C_6928498	net/minecraft/world/biome/Biome
 	f	Ljava/util/List;	f_4280813	waterEntries
 	f	Z	f_0598118	snowy
 	f	Z	f_9719651	hasRain
-	f	I	f_1682457	id
+	f	I	f_1682457	waterFogColor
```

Comment:
Just making sure since it's a far-reaching change, but was this field misnamed in 1.3 or was it mismatched and is this name only wrong in some later versions?

Reply:
as far as i know, wrong in 1.13. probably due to a mismatch as pre 1.13, IDs are essentially just ints, and since 1.13, IDs are Strings. I renamed this field by propagating from the toString method in the Biome.Builder class, so i am pretty sure at least for 1.13 this name is correct

Reply:
I see. I'll just leave it for now and investigate it for 1.12 and below later.